components/Profile/Forms/ChangePassword.vue

```diff
@@ -26,15 +26,53 @@
       </BaseButton>
     </form>
   </BaseCard>
+  <BaseModal
+    :is-shown="isShown"
+    @close="userCloseModal"
+  >
+    <template #title>Changed!</template>
+    <template #content>
+      <div class="text-center">You successfully changed your password!</div>
+      <div class="mt-5 flex justify-center">
+        <BaseButton
+          class="w-40"
+          @click="userCloseModal"
+        >
+          Okay, got it!
+        </BaseButton>
+      </div>
+    </template>
+  </BaseModal>
 </template>
 
 <script setup lang="ts">
+const { isLoading: isUpdateLoading, mutate: updateUser } = useUserUpdate();
+const { isShown, show: showModal, hide: hideModal } = useModal();
+
 const formData = ref({
   password: '',
-  newPassword: '',
+  passwordConfirmation: '',
 });
 
+const userCloseModal = () => {
+  if (!isUpdateLoading.value) {
+    hideModal();
+    formData.value.password = '';
+    formData.value.passwordConfirmation = '';
+  }
+};
+
 const saveInformation = () => {
-  console.warn(formData.value);
+  const { password, passwordConfirmation } = formData.value;
+
+  if (!(password || passwordConfirmation)) {
+    console.warn('No password provided.');
+  } else if (password !== passwordConfirmation) {
+    console.warn('Passwords are not equal.');
+  } else if (password.length < 6) {
+    console.warn('Your password should be at least 6 characters.');
```

Comment:
This validation is for testing purposes only, of course we have to implement something solid instead of this

Reply:
Oke 👌